### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by kneekey23**
update issue template for better context on bugs

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -14,23 +17,42 @@ Steps to reproduce the behavior:
...
 
-**Which AWS service(s) are affected?**
+**Observed Behavior**
 
-**Expected behavior**
+**Expected Behavior**
 A clear and concise description of what you expected to happen.
 
+**Stack Trace**
+Please provide a stack trace if applicable i.e. a crash is occurring.
```
Comment (wooj2) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: e.g.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -14,23 +17,42 @@ Steps to reproduce the behavior:
...
+
+**Code Snippet**
+Please provide a snippet of the code causing the issue or how you are using the service that has the bug.
+
+**Unique Config**
```
Comment (wooj2) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: s/Config/Configuration/

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
 ---
 name: Bug Report
 about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (royjit) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: What are these values for?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -14,23 +17,42 @@ Steps to reproduce the behavior:
...
 A clear and concise description of what you expected to happen.
 
+**Stack Trace**
+Please provide a stack trace if applicable i.e. a crash is occurring.
+
+**Code Snippet**
+Please provide a snippet of the code causing the issue or how you are using the service that has the bug.
+
+**Unique Config**
+If you are reporting an issue with a unique configuration or where configuration can make a difference in code execution (i.e. Cognito) please provide your configuration.
```
Comment (royjit) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Mention them to remove sensitive information from the configuration is required?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (kneekey23) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (kneekey23) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (kneekey23) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: the title of the issue and labels added. if we wanted any pre-added labels we could add them here.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (kneekey23) [reply .github/ISSUE_TEMPLATE/bug_report.md]: fixed

**Comment by lawmicha** [comment]
@kneekey23 all merges should go in `develop` branch btw

**Pull request merged**